### DISCUSSION TITLE
Mergear Pull Requests: Rebase ou Merge default?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1802] [Perguntas e Respostas]
Galera, 
estamos definindo nosso fluxo aqui na empresa (de nosso squad), e ficamos com esta dúvida, quando alguém manda um pull request dos branchs de features para a devlop, seguindo o gitflow, vocês fazem o merge como? Rebase? erge default? Quais as vantagens e desvantagens que veem em fazer assim?

**Comment by yuritoledo** [comment]
Merge default após abrir a PR mesmo.

O rebase é legal fazer antes de abrir o PR, pra não 'sujar' o histórico da master
